fluent-plugin-webhdfs.gemspec

```diff
@@ -17,7 +17,8 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency "rake"
   gem.add_development_dependency "test-unit"
-  gem.add_runtime_dependency "fluentd", '>= 0.10.53'
+  gem.add_development_dependency "appraisal"
+  gem.add_runtime_dependency "fluentd", '>= 0.10.59'
```

Comment:
Because `@type` notation is since Fluentd v0.10.59.
We use `@type` notation In README.md.